lfds-swagger.json

```diff
@@ -0,0 +1,239 @@
+{
+  "x-generator": "NSwag v13.14.8.0 (NJsonSchema v10.5.2.0 (Newtonsoft.Json v13.0.0.0))",
+  "openapi": "3.0.0",
+  "info": {
+    "title": "Laserfiche Repository API",
+    "description": "Welcome to the Laserfiche API Swagger Playground. You can try out any of our API calls against your live Laserfiche Cloud account. Visit the developer center for more details: <a href=\"https://developer.laserfiche.com\">https://developer.laserfiche.com</a><p><strong>Build# : </strong>develop_localbuild</p>",
+    "version": "1-alpha"
+  },
+  "servers": [
+    {
+      "url": "http://localhost:11211"
+    },
+    {
+      "url": "https://localhost:11211"
+    }
+  ],
+  "paths": {
+	"/v1-alpha/Repositories/{repoId}/AccessTokens/Create": {
+      "post": {
+        "tags": [
+          "AccessTokens"
+        ],
+        "description": "- Creates an access token for use with the Laserfiche API.\n- Provides credentials and uses the access token returned with subsequent API calls as a means of authorization.\n- Adding createCookie=true as a query parameter results a response that includes a Set-Cookie header containing an authToken value. The default value for createCookie is false.",

```

Comment:
could you change the `description` property to `summary`? This will allow the autogenerated client to have the xml documentation.